inst/include/Rcpp/api/meat/module/Module.h

```diff
@@ -23,12 +23,12 @@
 namespace Rcpp {
 
     inline List Module::classes_info(){
-	    int n = classes.size() ;
+	    size_t n = classes.size() ;
```

Comment:
@kevinushey the `size_t` changes in this PR are mostly like this. No access from users.